content/_releases/22/50.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Product Update - v22.50
+description: Details of the product update v22.50 from 15th of December 2022.
+layout: rel
+version: "22.50"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-12-15
+redirect_from:
+  - /releases/22.50.html
+---
+
+## Features
+
+### Recipes list: Use Interface
+
+In this release we introduce a new dedicated page for recipe management accessible
+through new Recipes menu item available under Integrate section. Now all recipe
+deployments are listed here similar to the flows page.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2250/rn2250-recipes.png" title="Recipes" %}
+
+You can use all listed recipes to create an integration flow by pressing
+the **Activate** button. If the recipe is already used in a flow deployment,
```

Comment:
If the recipe has already been used to deploy a flow, the counter for **Recipe Deployments** will indicate the number of flows deployed using this recipe.

Reply:
@KMobbs it's not correct. It will show the number of deployments. And each deployment may contain several flows
![image](https://user-images.githubusercontent.com/28867440/207833551-d2438218-b51c-46e2-add8-6f39bc2b00e9.png)
![image](https://user-images.githubusercontent.com/28867440/207833644-e5571863-4102-4877-aa11-6323bb9fea9b.png)

Reply:
OK so I will write the number of recipe deployments.